source/opprette-oppdrag.rst

```diff
@@ -74,7 +74,7 @@ For testing, use the example documents :download:`PDF-1.2 <files/PDF-1-2-testdok
 In PAdES, the documents will always be presented in A4 and portrait format. For best results, it is recommended that the submitted documents are also in this format.
 
 ..  CAUTION::
-    Password-protected documents (restricted reading and/or writing access) are not supported by the service and will not show an error message until the document is downloaded.
+    Password-protected documents (restricted reading and/or writing access) are not supported by the service. When using the API, an error message will not be shown until the signed document is downloaded.
```

Comment:
Dette tror jeg rett og slett er feil 🤯 
Vi validerer at PDF-er ikke er passordbeskyttet allerede ved opprettelse av oppdrag, både via API og avsenderportal.
https://github.com/digipost/signering/blob/main/app/src/main/java/no/posten/signering/common/signaturejob/validation/DocumentValidator.java#L57-L62

Reply:
Altså feilen er at man ikke får feilmelding før signert dokument lastes ned. Man får ikke opprettet noe oppdrag med passordbeskyttet og/eller krypterte PDF-er.

Reply:
Se der ja, visste at vi gjorde det for portal så derfor jeg endret på det, men da sletter jeg heller hele setningen. Trenger vi å skrive at `Password-protected documents (restricted reading and/or writing access) are not supported by the service` da eller kan vi bare drite i det fordi folk får beskjed ved opprettelse uansett?

Reply:
Tja, jeg tenker det ikke gjør noe at det står der. Greit å ha det dokumentert, og å få en heads-up på det når man leser dokumentasjon for å ta i bruk tjenesten. (Kanskje først og fremst for API-integrasjon, fordi man får det forklart i avsenderportalen)